src/main/resources/en/internet.yml

```diff
@@ -15,7 +15,7 @@ en:
       - "Visit #{Country.name}"
       - "Want to play a game?"
       - "Since we can’t all win the lottery..."
-      - "#{FamousLastWords}"
+      - "#{FamousLastWords.lastWords}"
```

Comment:
This is a dangerous usage
it could potentially change the behavior once there will be more methods added
that's why there should be a specific method mentioned